g11n-ws/docs/developer_guide_javaclient.md

```diff
@@ -0,0 +1,25 @@
+## Developer's Guide for Singleton Java Client
+
+### How to setup Intellij IDEA for development
+
+You can follow below steps to setup your development environment:
+
+1. Install JDK 1.8(or later version) and Gradle 4.10.3(or later version)
+
+2. Install Intellij IDEA community version and configure Gradle home
+
+3. Import Java client codes as Gradle project
+
+4. Run build.gradle with 'clean build' to generate the build
+
+After building, you can find the build of Singleton's Java client from the building output folder. 
```

Comment:
The path is under the project '/build/libs/'

Reply:
fixed